src/hooks/useTheme.ts

```diff
@@ -0,0 +1,16 @@
+import React, { useContext } from "react";
+import defaultTheme from "../themes/default";
+import { EnhancedTheme } from "../themes/types";
+import { enhanceTheme } from "../utils/theme";
+
+export const ThemeContext = React.createContext<EnhancedTheme>(
+  enhanceTheme(defaultTheme)
+);
+
+function useTheme() {
+  const theme = useContext(ThemeContext);
```

Comment:
can't we just return the `useContext(ThemeContext)` without creating the theme variable?